src/RefreshOptions.ts

```diff
@@ -25,3 +25,17 @@ export interface RefreshOptions {
      */
     watchedSettings?: WatchedSetting[];
 }
+
+export interface FeatureFlagRefreshOptions {
+    /**
+     * Specifies whether the provider should automatically refresh when the configuration is changed.

```

Comment:
```suggestion
     * Specifies whether the provider should automatically refresh all configurations if any feature flag changes.

```

Reply:
Just to confirm - we're not going to refresh key-values with FeatureFlag changes and FF with kv changes, right? Both refreshes are done independently of each other?